src/Microsoft.DotNet.Build.VstsBuildsApi/VstsReleaseHttpClient.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.VstsBuildsApi
+{
+    internal class VstsReleaseHttpClient : IVstsDefinitionHttpClient
+    {
+        private static readonly string[] s_InstanceIdentifiableField = {
+            "_links",
+            "createdBy",
+            "createdOn",
+            "deployStep",
+            "modifiedBy",
+            "modifiedOn",
+            "owner",
+            "id",
+            "lastRelease",
+            "revision",
+        };
```

Comment:
I wonder if we should move these (and the ApiVersion) outside of the classes and into an external file, I guess it's not clear how / if this information would change in a different api version, but editing a config file would be preferable to recompiling or ensuring you had the right toolset for the right api version.